### ISSUE TITLE
feat: downloads page

### ISSUE THREAD

**Issue by Axelen123** [revanced/revanced-website#20]
We are going to want this.

**Comment by Abhinav1217** [comment]
Since revanced doesn't actually provide downloads, I would prefer the website to be a kind of documentation for developers, and patch listing for general users. Maybe a tutorials on Patch building and reverse engineering. A discussion on yt culture ytubers that the algorithm are not promoting enough.

And a showcase for associated community projects like revanced builder, rvx etc (staying within the foss realm and not providing compiled binaries.  

In short, more of a community website than a website that could make google think that we are doing same thing like vanced.

**Comment by Axelen123** [comment]
> Since revanced doesn't actually provide downloads, I would prefer the website to be a kind of documentation for developers, and patch listing for general users. Maybe a tutorials on Patch building and reverse engineering. A discussion on yt culture ytubers that the algorithm are not promoting enough.
> 
> And a showcase for associated community projects like revanced builder, rvx etc (staying within the foss realm and not providing compiled binaries.
> 
> In short, more of a community website than a website that could make google think that we are doing same thing like vanced.

ReVanced will never provide APK downloads for patched apps, because that is exactly why the Vanced project got a cease and desist letter. 
The downloads page will mainly have ReVanced Manager, which patches the apps on the users device instead.

**Comment by oSumAtrIX** [comment]
Can you clarify by what you mean with downloads page. What do you plan on offering as a download?

**Comment by Axelen123** [comment]
Mainly ReVanced Manager
